.github/workflows/testing.yml

```diff
@@ -58,4 +58,6 @@ jobs:
         run: make install
 
       - name: Run test suite
+        env:
```

Comment:
Nic asks: is this still relevant?

Reply:
answer to self: nope

db73b46df09c77f09a5034daced8d358f1c50628